app/views/whitehall_migration/document.html.erb

```diff
@@ -1,2 +1,27 @@
-<h1>Document</h1>
-<%= @document.id %>
+<% content_for :back_link, render_back_link(href: whitehall_migration_documents_path(@document.whitehall_migration)) %>
+<% content_for :title, "Document Import: #{@document.id}" %>
+<%= render "govuk_publishing_components/components/summary_list", {
+  items: [
+    {
+      field: "ID",
+      value: @document.id
+    },
+    {
+      field: "Time started",
+      value: format_time_on_date(@document.created_at)
+    },
+    {
+      field: "Time finished",
+      value: @document.whitehall_migration.finished_at.nil? ? "unfinished" : format_time_on_date(@document.whitehall_migration.finished_at)
+    },
+    {
+      field: "Status",
+      value: @document.state
+    },
+    {
+      field: "Error log",
+      value: @document.error_log
+    }
```

Comment:
We'll likely want to also include integrity check problems here.

May even want to dump things like payload, integrity check problems payload and the list of asset imports. Though they can come later.

Reply:
@kevindew that would be good. Sounds like one for a future PR?